llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2946,7 +2947,8 @@ class VPBasicBlock : public VPBlockBase {
 private:
   /// Create an IR BasicBlock to hold the output instructions generated by this
   /// VPBasicBlock, and return it. Update the CFGState accordingly.
-  BasicBlock *createEmptyBasicBlock(VPTransformState::CFGState &CFG);
+  BasicBlock *createEmptyBasicBlock(VPTransformState::CFGState &CFG,
```

Comment:
nit: if DTU is part of CFGState, then "Update the CFGState accordingly" would also imply updating DT, which this method now does.

Reply:
Done, thanks!